backend/api/communication/src/main/java/co/airy/core/api/communication/SendMessageController.java

```diff
@@ -105,4 +108,26 @@
 
         return ResponseEntity.status(s).body(MessageResponsePayload.fromMessageContainer(new MessageContainer(message, new MetadataMap())));
     }
+
+    @PostMapping("/messages.resend")
+    public ResponseEntity<?> sendMessage(@RequestBody @Valid ResendMessageRequestPayload payload) {
+        final String messageId = payload.getMessageId().toString();
+
+        final MessageContainer messageContainer = stores.getMessageContainer(messageId);
+        if (messageContainer == null) {
+            return ResponseEntity.status(HttpStatus.NOT_FOUND).build();
```

Comment:
Just to be aware the in the case that the conversation is never ready. The message won't be ingested so this will always return `404`